src/overrides/venia-ui/components/ProductFullDetail/productFullDetail.js

```diff
@@ -222,18 +222,15 @@ const ProductFullDetail = props => {
     const tabsData = [
         {
             tabTitle: "Description",
-            textContentTitle: "title:Product Description",
-            content: productDetails.shortDescription.html ? productDetails.shortDescription.html.replace(/<p>|<\/p>/g, "").trim() : "No short description found"
-            
-            
+            content: [<h1 className={classes.tabTitle}>Product Description</h1>, productDetails.shortDescription.html ? <RichContent html={productDetails.shortDescription.html} /> : <RichContent html={"<p>No short description found</p>"} />]
         },
         {
             tabTitle: "Attributes",
```

Comment:
Also for any text that you use in the app that is user facing. You should use <FormattedMessage> or `const { formatMessage } = useIntl()` (depending on context) to make them translatable.